blueocean-pipeline-api/src/main/java/io/jenkins/blueocean/pipeline/creation/AbstractMultiBranchCreateRequest.java

```diff
@@ -0,0 +1,132 @@
+package io.jenkins.blueocean.pipeline.creation;
```

Comment:
@vivek sounds reasonable. Ive made the change. I also moved the credentials back into `io/jenkins/blueocean/rest/impl/pipeline/credential` package as I think they their fully qualified names get baked into xstream